src/Assets/Javascript/toggle.js

```diff
@@ -0,0 +1,25 @@
+// Toggle element visibility
+var toggle = function (elem) {
+  elem.classList.toggle('is-visible');
+};
+
+// Listen for click events
+document.addEventListener('click', function (event) {
```

Comment:
Can the JS also add `aria` attributes when it first executes, and then update them when toggled to represent state?